com.unity.render-pipelines.high-definition/Runtime/Material/GGXConvolution/ComputeGgxIblSampleData.compute

```diff
@@ -5,20 +5,15 @@
 
 #pragma only_renderers d3d11 playstation xboxone vulkan metal switch
 
-#if defined(SHADER_API_MOBILE) || defined(SHADER_API_SWITCH)
-#define MAX_IBL_SAMPLE_CNT 34
-#else
-#define MAX_IBL_SAMPLE_CNT 89
-#endif
-
 RWTexture2D<float4> outputResult; // [MAX_SAMPLE_CNT x UNITY_SPECCUBE_LOD_STEPS]
 
-#pragma kernel ComputeGgxIblSampleData
+#pragma kernel ComputeGgxIblSampleData COMPUTEGGGXIBLSAMPLEDATA=ComputeGgxIblSampleData MAX_IBL_SAMPLE_CNT=89
```

Comment:
I guess I have a global misunderstand about what "LE" mean. it is just to have 32 vs 64? Or does it mean Lightweight?

here it is not a matter of 32 vs 64 thread size, it is for performance. Mobile and switch don't have the power to process the 89 sample of PC / console, so we use a lower number of sample.

We shouldn't change this.

Reply:
For future Macs, we need both of these to be available based on GPU used during runtime.